content/c-sharp/concepts/strings/terms/substring/substring.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: '.Substring()'
+Description: 'Returns the substring of a string instance starting at a given index'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Substring()`** method is a string method that returns a substring of a string starting at the given index number, it will return all characters unless a maximum length is given. This method returns `Empty` if the index is greater than the length of the string instance.
+
+## Syntax
+
+```pseudo
+Substring(int startIndex);
+Substring(int startIndex, int Length)
```

Comment:
We can write this as:

.Substring(int startIndex)

Or, alternatively:

.Substring(int startIndex, int length)